test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13740,6 +13740,15 @@ def create_subjob_id(self, job_array_id, subjob_index):
         return job_array_id[:idx + 1] + str(subjob_index) + \
             job_array_id[idx + 1:]
 
+    def create_eatcpu_job(self, duration=None):
+        """
+        Create a job that eats cpu indefinitely or for the given
+        duration of time
+        """
+        script_dir = os.path.dirname(os.path.dirname(__file__))
+        script_path = os.path.join(script_dir, 'eatcpu.pl')
+        self.set_execargs(script_path, duration)
+
```

Comment:
After reviewing the entire test, there is no need for any of these jobs to eat cpu time.  Sleep jobs will work just fine.  There is no need to enhance PTL to create jobs that eat cpu time.  Just use the default sleep jobs.

Reply:
Yes, but why does that matter?  These tests are testing for the existence of 'R' records.  It doesn't matter if the jobs eat 30s of cpu time or they sleep for 30s.  The 'R' records will still exist.

Reply:
I thought some of these tests checked the resource usage.
I still think it's useful to have the script in ptl, but for these tests we don't need it.

Reply:
Yes.  I do prefer to keep the script and it would be helpful for tests that compare resources_usage values in accounting logs.  I haven't automated one such test in this PR though.

Reply:
This is OK, but I'd rather see you reimplement this script in python.  PTL is written in python, so I don't see a reason to start using perl now.